data/config/serverdb.kvc

```diff
@@ -176,6 +176,14 @@ NServers=10
 1_Port=6697
 1_SSL=true
 NServers=2
+[darkscience]
+0_Hostname=irc.darkscience.net
+0_Description=darkscience:%20Main%20address
+1_Hostname=irc.drk.sc
```

Comment:
I don't see reference to this one on their website.

Reply:
you can see it in their 'Subject Alternative Names':

```
$ openssl s_client -connect irc.drk.sc:6697 2>/dev/null | openssl x509 -noout -text | grep -A1 'Subject Alternative Name'

            X509v3 Subject Alternative Name:
                DNS:irc-us-east-1.darkscience.net, DNS:irc.darkscience.net, DNS:irc.darkscience.org, DNS:irc.darkscience.ws, DNS:irc.drk.sc, DNS:irc.hak5.org, DNS:irc.vhirc.net
```

Reply:
You didn't add other Alts from the cert though. Is the name advertised anywhere other than inside the cert?

Reply:
The alt name is surely legit; also @dijit seems to be a network admin, so i guess it's safe to trust him